src/test/fuzz/coins_view.cpp

```diff
@@ -41,13 +42,12 @@ void initialize_coins_view()
     static const auto testing_setup = MakeNoLogFileContext<>();
 }
 
-FUZZ_TARGET(coins_view, .init = initialize_coins_view)
+void TestCoinsView(FuzzedDataProvider& fuzzed_data_provider, CCoinsView& backend_coins_view, bool is_db = false)
```

Comment:
The default value for `is_db` may not be intuitive, can we make it explicit instead?